doc/Readme.md

```diff
@@ -42,6 +43,43 @@ sudo ./opentap.linux --quiet
 The installer is likely to work on other Linux distributions, but additional dependencies
 may be required on these platforms, such as dotnet 6 runtime.
 
+#### Other Linux distributions
+For other Linux distributions, OpenTAP can be installed using the terminal:
+
+1. Install dotnet: [Instructions](https://learn.microsoft.com/da-dk/dotnet/core/install/linux?WT.mc_id=dotnet-35129-website).
+    - For Ubuntu: `sudo apt install -y dotnet-sdk-8.0`
+2. Download the latest OpenTAP release: `curl -Lo opentap.zip 'https://packages.opentap.io/4.0/Objects/Packages/OpenTAP?os=Linux&architecture=<architecture>'`
+    - Remember to change the architecture if needed (`x64`, `arm64`, `arm`).
+3. Create a directory for OpenTAP: `sudo mkdir /opt/opentap && sudo chown $USER:$USER /opt/opentap`
+4. Extract the OpenTAP release: `unzip ./opentap.zip -d /opt/opentap`
```

Comment:
It is pretty weird to suggest a single user installation in /opt/. If an application is not inside the user's home folder, it should be a multi-user installation. A single user application should always be in the user's home folder. Most common is something like `/home/$USER/.local/share/opentap`.